mv.sql

```diff
@@ -746,3 +746,75 @@ WHERE slug IN (
                'aiguille-du-dru',
                'aiguille-de-bionnassay'
     );
+
+
+
+-- Aporte de Cristian Morano desde aquí
+
+-- Montes gallegos
+-- Insert challenge
+INSERT INTO challenge (id, name, slug, country)
+VALUES ('01961783-e221-7dbb-8c14-99a4377d3c9e', 'Camiño das Cimas', 'montes-gallegos', 'ESP');
+
+-- Insert mountains
+INSERT INTO mountain (slug, name, location, height, latitude, longitude, essential, image_url)
+VALUES
+    ('pena-trevinca', 'Peña Trevinca', 'Valdeorras - Sierra del Eje', 2127, 42.2424, -6.7960, TRUE, 'https://i.imgur.com/Sjifivk.jpg')
+    ('pena-negra', 'Peña Negra', 'Valdeorras - Sierra del Eje', 2121, 42.2495, -6.7942, TRUE, 'https://i.imgur.com/tEFWLqC.jpg')
```

Comment:
Won't work either, is `.jpeg` not `.jpg`.

As a rule of thumb, you must be able to open the image and only see the image:
- https://i.imgur.com/Sjifivk.jpg -> opens imgur app
- https://i.imgur.com/Sjifivk.jpeg -> opens the image